ts/Series/Treegraph/TreegraphLink.ts

```diff
@@ -0,0 +1,68 @@
+import type PointOptions from '../../Core/Series/PointOptions.js';
+import type TreegraphSeries from './TreegraphSeries';
+import type { LinkOptions } from '../Organization/OrganizationSeriesOptions.js';
+import Point from '../../Core/Series/Point.js';
+import TreegraphPoint from './TreegraphPoint.js';
+import TreegraphPointOptions from './TreegraphPointOptions.js';
+import U from '../../Core/Utilities.js';
+const { pick, extend } = U;
+
+export interface LinkPointOptions extends TreegraphPointOptions {
+    link?: LinkOptions
+}
+class LinkPoint extends Point {
+    options: LinkPointOptions = void 0 as any;
+    fromNode: TreegraphPoint = void 0 as any;
+    toNode: TreegraphPoint = void 0 as any;
```

Comment:
You can move these properties to class prototype interface to avoid the any assignment.